MediaBrowser.Controller/Drawing/ImageProcessingOptions.cs

```diff
@@ -95,6 +99,16 @@ public bool HasDefaultOptions(string originalImagePath, ImageDimensions? size)
                 return false;
             }
 
+            if (FillWidth.HasValue && sizeValue.Width > FillWidth.Value)
+            {
+                return false;
+            }
+
+            if (FillHeight.HasValue && sizeValue.Height > FillHeight.Value)
```

Comment:
Comparisons with null return false.
```suggestion
            if (sizeValue.Width > FillWidth || sizeValue.Height > FillHeight)
```